nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -658,13 +691,17 @@ in
       '';
     };
 
-    isoImage.graphicalGrub = mkOption {
```

Comment:
`mkRemovedOptionModule` might be a good idea

Reply:
is isoImage part of the public interface?

Reply:
It's not really part of the public interface (it requires importing the module) and it was never in a stable release. So changing that option outright without an `mkRemovedOptionModule` should be fine.

Though I'm not against doing it, if deemed necessary.